barretenberg/cpp/src/barretenberg/sumcheck/sumcheck.hpp

```diff
@@ -403,7 +408,9 @@ polynomials that are sent in clear.
         EvalMaskingScalars eval_masking_scalars;
 
         for (size_t k = 0; k < NUM_ALL_WITNESS_ENTITIES; ++k) {
-            eval_masking_scalars[k] = FF::random_element();
+            // TODO(https://github.com/AztecProtocol/barretenberg/issues/1136): Once Shplemini supports ZK, these
```

Comment:
this is only if we do something smarter than generating a circuit_size random polynomials.